tests/apertium/ApertiumLintBearTest.py

```diff
@@ -0,0 +1,48 @@
+import os
+from queue import Queue
+
+from bears.apertium.ApertiumLintBear import ApertiumLintBear
+from coalib.testing.BearTestHelper import generate_skip_decorator
+from coalib.testing.LocalBearTestHelper import verify_local_bear
+from coalib.testing.LocalBearTestHelper import LocalBearTestHelper
+from coalib.settings.Section import Section
+from coalib.settings.Setting import Setting
+
+test_files = os.path.join(os.path.dirname(__file__), 'test_files')
+good_file = os.path.join(test_files, 'apertium-go-od.en.dix')
+bad_file = os.path.join(test_files, 'apertium-ba-ad.en.dix')
+
+ApertiumLintBearDefaultTest = verify_local_bear(
+    ApertiumLintBear,
+    valid_files=(),
+    invalid_files=[],
+    filename=bad_file)
+
+ApertiumLintBearParadigmTest = verify_local_bear(
+    ApertiumLintBear,
+    valid_files=[],
+    invalid_files=(),
+    filename=bad_file,
```

Comment:
It is incredibly easy to prove this is doing nothing as I previously indicated.

Add an `assert False` in `ApertiumLintBear.create_arguments`, and then run the tests.  These two test classes will pass, and `test_run` will fail.

```
requests/github/1438$ git diff
diff --git a/bears/apertium/ApertiumLintBear.py b/bears/apertium/ApertiumLintBear.py
index 3fbbc8f..cfa6218 100644
--- a/bears/apertium/ApertiumLintBear.py
+++ b/bears/apertium/ApertiumLintBear.py
@@ -211,6 +211,7 @@ class ApertiumLintBear:
             the name should have an appropriate suffix like <U+00AD>morph,
             interchunk, etc.
         """
+        assert False
         monodix = {'redundantPardef': {'message': 'Warning : Redundant pardef '
                                        'entry present for :',
                                        'enable': redundant_pardef},

$ python3 -m pytest tests/apertium/ApertiumLintBearTest.py -k ApertiumLintBearParadigmTest
============ test session starts ============
platform linux -- Python 3.6.1, pytest-3.0.7, py-1.4.33, pluggy-0.4.0
rootdir: /home/jayvdb/projects/coala/coala-bears, inifile: setup.cfg
plugins: xdist-1.15.0, timeout-1.2.0, mock-1.5.0, env-0.6.0, cov-2.4.0
timeout: 35.0s method: signal
collected 5 items 

tests/apertium/ApertiumLintBearTest.py::ApertiumLintBearParadigmTest::test_invalid_files <- ../coala/coalib/testing/LocalBearTestHelper.py PASSED
tests/apertium/ApertiumLintBearTest.py::ApertiumLintBearParadigmTest::test_valid_files <- ../coala/coalib/testing/LocalBearTestHelper.py PASSED


============ 3 tests deselected ============
```